src/config.ts

```diff
@@ -91,109 +91,6 @@ export interface NetworkInfo extends Network {
-    decimals: 18,
-    contracts: {
-      ledger: '0x3196f354b7a95413E30889D1C6cE5074b10c43f5',
-      winPay: '0x6f2fBD652A99Db4b8143c8383Ae39b5459268685',
-      assets: [
-        {
-          name: 'Native xDAI',
-          symbol: 'xDAI',
-          address: '0x25149dE5afe2043C61687AD136527d2167EFC241',
-          decimals: 18,
-          image:
-            'https://bafybeiesj7lzhl7gb3xnnazkozdh6cdsby2nmgphqc6ts6rnlf4mnczzbm.ipfs.dweb.link/8635.png',
-          native: true,
-          permit: false,
-          currency: 'USD'
-        },
-        {
-          name: 'Wrapped xDAI',
-          symbol: 'wxDAI',
-          address: '0x25149dE5afe2043C61687AD136527d2167EFC241',
-          decimals: 18,
-          image:
-            'https://bafybeicj27bao6jkip26yhvc32tcyror5asop6dfxk3db67yfkxc6me6ym.ipfs.dweb.link/9021.png',
-          native: false,
-          permit: true,
-          currency: 'USD'
-        },
-        {
-          name: 'USDC',
-          symbol: 'USDC',
-          address: '0x1C375919362730AC69c5ACffeC775F11c9b75cF2',
-          decimals: 18,
-          image:
-            'https://bafybeif5mtgb4mtvvqbhw2kdr4uruu5xm742vtwa3cwndpnsqdb2t4676m.ipfs.dweb.link/3408.png',
-          native: false,
-          permit: true,
-          currency: 'USD'
-        }
-      ]
-    }
-  }
-];
-if (allowHardhatNetwork) {
-  const localNetwork = {
-    name: 'Localhost',
-    chainId: 31337,
-    rpc: 'http://127.0.0.1:8545',
-    blockExplorer: '',
-    currency: 'xDAI',
-    decimals: 18,
-    contracts: {
-      ledger: '0x5FC8d32690cc91D4c39d9d3abcBD16989F875707',
-      winPay: '0x610178dA211FEF7D417bC0e6FeD39F05609AD788',
-      assets: [
-        {
-          name: 'Native xDAI',
-          symbol: 'xDAI',
-          address: '0x2279B7A0a67DB372996a5FaB50D91eAA73d2eBe6',
-          decimals: 18,
-          image:
-            'https://bafybeiesj7lzhl7gb3xnnazkozdh6cdsby2nmgphqc6ts6rnlf4mnczzbm.ipfs.dweb.link/8635.png',
-          native: true,
-          permit: false,
-          currency: 'USD'
-        },
-        {
-          name: 'Wrapped xDAI',
-          symbol: 'wxDAI',
-          address: '0x2279B7A0a67DB372996a5FaB50D91eAA73d2eBe6',
-          decimals: 18,
-          image:
-            'https://bafybeicj27bao6jkip26yhvc32tcyror5asop6dfxk3db67yfkxc6me6ym.ipfs.dweb.link/9021.png',
-          native: false,
-          permit: true,
-          currency: 'USD'
-        },
-        {
-          name: 'USDC',
-          symbol: 'USDC',
-          address: '0x0165878A594ca255338adfa4d48449f69242Eb8F',
-          decimals: 18,
-          image:
-            'https://bafybeif5mtgb4mtvvqbhw2kdr4uruu5xm742vtwa3cwndpnsqdb2t4676m.ipfs.dweb.link/3408.png',
-          native: false,
-          permit: true,
-          currency: 'USD'
-        }
-      ]
-    }
-  } as NetworkInfo;
-
-  networks.push(localNetwork);
-}
-
-export const allowedNetworks: readonly NetworkInfo[] = Object.freeze(networks);
-
 const encodedWallet =
```

Comment:
we need to mark this definition as dedicated to testing, otherwise, it looks strange (hardcoded wallet in the app, hmm...)